### DISCUSSION TITLE
master/slave config

### DISCUSSION THREAD

**Discussion by sartoricz** [oetiker/SmokePing#382] [General]
When in master/slave scenario, is there possible to monitor host only from slaves and not from the master? With the following config example,. host gets monitored from both master and slave.

```
+ test

menu = test
title = test
slaves = slave-1

++ some.host.tld

menu = ip.address
title = ip.address
host = ip.address
```

**Comment by moetiker** [comment]
You can use the option 
`nomasterpoll = yes`

**Reply by sartoricz** [reply]
@moetiker thank you, this solves my problem
